pkg/querier/tenantfederation/merge_queryable.go

```diff
@@ -170,14 +170,12 @@ func (m *mergeQuerier) LabelValues(ctx context.Context, name string, matchers ..
 	spanlog, ctx := spanlogger.NewWithLogger(ctx, m.logger, "mergeQuerier.LabelValues")
 	defer spanlog.Finish()
 
-	matchedTenants, filteredMatchers := filterValuesByMatchers(m.idLabelName, ids, matchers...)
+	matchedIDs, filteredMatchers := filterValuesByMatchers(m.idLabelName, ids, matchers...)
```

Comment:
Calling it `matchedIDs` is consistent with the naming of the `ids` variable. `matchedIDs` is just a filtered version of `ids`.